tests/integration/cli/test_test.py

```diff
@@ -13,6 +13,7 @@
 
   Method +Times called +Min. +Max. +Mean +Median
  ─+
+  __init__ +\d +\d+ + \d+ + \d+ + \d+
```

Comment:
I'm not sure what these do

Reply:
it is regex that ensures the method has numeric values for all the gas columns with some spacing in between